cmd/fossa/main.go

```diff
@@ -131,6 +131,19 @@ func main() {
 				cli.BoolFlag{Name: "debug", Usage: debugUsage},
 			},
 		},
+		{
+			Name:   "report",
+			Usage:  "Generates a license report",
+			Action: reportCmd,
+			Flags: []cli.Flag{
+				cli.StringFlag{Name: "c, config", Usage: configUsage},
```

Comment:
fetcher?

Reply:
Good catch. We're going to switch these to use global flags.